desktop/wallet/package.json

```diff
@@ -26,7 +26,7 @@
     "node": ">=10"
   },
   "scripts": {
-    "build": "node ../../scripts/build.js --target desktop",
+    "build": "cross-env NODE_OPTIONS=--max_old_space_size=4096 node ../../scripts/build.js --target desktop",
```

Comment:
this is the CI OOM fix

Reply:
do we need to add cross-env to package.json?